src/App.js

```diff
@@ -11,13 +11,21 @@ function App() {
 
  const [contacts, setContact] = useState(arrOfFive);
 
+ 
 
+  const addRandomContact = () => {
+    
+    let filterList = contactsArr.filter((newContact) => {
+      return !contacts.includes(newContact)
+      })
```

Comment:
Always make a copy of the array. Get into habit of working on copied data rather than data you have received from the database, or original source. 

You can simply do
```
let filterList = [...contactsArr].filter
```